src/main/plugin/iso19139.rndt/update-fixed-info.xsl

```diff
@@ -231,6 +233,19 @@
     <!-- ================================================================= -->
     <!-- CI_Series -->
 
+  <xsl:template match="gmd:identificationInfo/*/gmd:citation/gmd:CI_Citation[not(gmd:series)]" >
```

Comment:
Se viene aggiunto lo snippet da bottone + (primo file in questa PR) `gmd:series` eisste ma non è valorizzato, per cui questo template non matcha e il campo rimane vuoto

Reply:
fixed